### PR TITLE
Set up github actions pipeline

### PR DESCRIPTION
**Pull request by AndrewSavelyev**
Actually, subj )

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Dockerize API
```
Comment (michael-l-a) [inline .github/workflows/pipeline.yml]: Name should not be the commit content. It should reflect what this pipeline actually does

File: .github/workflows/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+on:
+  push:
+    branches: [ master ]
```
Comment (michael-l-a) [inline .github/workflows/pipeline.yml]: try to build it on all branches, don't limit it to master only

File: .github/workflows/pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Publish to Docker hub
+        uses: docker/build-push-action@v1
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: 0.1.0
```
Comment (michael-l-a) [inline .github/workflows/pipeline.yml]: tag must be calculated dynamically. Create a __init__.py file that has __version__ variable and use it

File: .github/workflows/pipeline.yml (hunk 2 above)
Comment (AndrewSavelyev) [reply .github/workflows/pipeline.yml]: ok

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (AndrewSavelyev) [reply .github/workflows/pipeline.yml]: Ok, will fix it

File: __init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = '1.0.1'
```
Comment (michael-l-a) [inline __init__.py]: no new line after this one. PEP8 issue

File: config/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+version = '1.0.1'
```
Comment (michael-l-a) [inline config/__init__.py]: 1. no new line after this one. PEP8 issue
2. use underscores surrounding the word.

File: .github/workflows/pipeline.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+name: Set up github actions
```
Comment (michael-l-a) [inline .github/workflows/pipeline.yml]: Is that a name of the pipeline? why?

File: .github/workflows/pipeline.yml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Get Application vars
+        run: |
+          source variables.sh
```
Comment (michael-l-a) [inline .github/workflows/pipeline.yml]: sourcing empty file?

**Review by michael-l-a — CHANGES_REQUESTED** [review]

File: .github/workflows/pipeline.yml (hunk 7 above)
Comment (AndrewSavelyev) [reply .github/workflows/pipeline.yml]: Already delete it

File: .github/workflows/pipeline.yml (hunk 6 above)
Comment (AndrewSavelyev) [reply .github/workflows/pipeline.yml]: So, maybe more correctly would be if i will set name as "Set up pipeline"?

File: config/__init__.py (hunk 5 above)
Comment (AndrewSavelyev) [reply config/__init__.py]: Ok. Should i left this file or delete it. I mean, this variable i set up in ./__init__.py. So file config/__init__.py must be clean, as i understood. Am i right?

File: __init__.py (hunk 4 above)
Comment (AndrewSavelyev) [reply __init__.py]: Ok, will add new line in this file

**Review by michael-l-a — APPROVED** [review]

**Pull request merged**